frontend/src/scenes/insights/InsightTabs/FunnelTab/FunnelCorrelationTable.tsx

```diff
@@ -353,6 +355,17 @@ const CorrelationActionsCell = ({ record }: { record: FunnelCorrelation }): JSX.
                 onClickOutside={() => setPopoverOpen(false)}
                 overlay={
                     <>
+                        {record.result_type === FunnelCorrelationResultsType.Events && (
+                            <LemonButton
+                                onClick={() =>
+                                    setFilters({ funnel_correlation_details: { key: components[0], type: 'event' } })
```

Comment:
Making this a `setFilter` call implies we send another request to recalculate the funnel, and then the correlations, which shouldn't happen here.

Reply:
Alright after some sweat and tears because my computer is going crazy with running CH, I managed to change this to use `hashParams` instead, this way we won't cause a reload anymore.